dali/pipeline/operator/op_schema.h

```diff
@@ -83,7 +83,7 @@ If not provided, it will be populated based on the global seed of the pipeline.)
     AddOptionalArg("bytes_per_sample_hint", R"code(Output size hint, in bytes per sample.
 
 If specified, the operator's outputs residing in GPU or page-locked host memory will be preallocated
-to accommodate a batch of samples of this size.)code", 0);
+to accommodate a batch of samples of this size.)code", std::vector<int>{0, });
```

Comment:
Nothing wrong with it, but is there a reason here to use trailing comma?

Reply:
I copy-paste it from other place. I can change.

Reply:
Done